tensorflow/contrib/model_pruning/python/pruning.py

```diff
@@ -520,7 +520,7 @@ def add_pruning_summaries(self):
       thresholds = get_thresholds()
       for index, mask in enumerate(masks):
```

Comment:
Shouldn't this be `for mask, threshold in zip(masks,thresholds):`

(optional)

Reply:
This would indeed be better, since `masks` and `thresholds` are always the same length.